website/docs/components/form/layout/partials/code/component-api.md

```diff
@@ -0,0 +1,200 @@
+  </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of the element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Form::Header
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="<[FH].Title>" @type="yielded component">
+    `Form::Header::Title` yielded as contextual component (see below).
+  </C.Property>
+  <C.Property @name="<[FH].Description" @type="yielded component">
+    `Form::Header::Description` yielded as contextual component (see below).
+  </C.Property>
+  <C.Property @name="isFullWidth" @type="boolean" @default="false">
+    Indicates that the header should take up the full width of the parent container.
+  </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of the element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Form::Header::Title
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="tag" @type="enum" @values={{array "div" "h1" "h2" "h3" "h4" "h5" "h6"}} @default="div">
+    A valid HTML tag name to be used to render the `Title` element.
+  </C.Property>
+  <C.Property @name="size" @type="string" @values={{array "500" "400" "300" "200" "100" }} @default="200">
+    The size of the `Display` text style.
+  </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of the element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Form::Header::Description
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of the element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Form::Section
+
+ <!-- 
+ Question: I included a yielded Section component within the Section to allow for a nested Section but does that actually make sense to include?
+ Should I keep the yielded Section within a Section and if so should I document it here? (I included it as sort of an edge case but I don't think we want to recommend or encourage that usage.)
+  -->
```

Comment:
@didoo I added this question here in a comment so I wouldn't forget to address it. What do you think?

```
Question: I included a yielded Section component within the 
Section to allow for a nested Section but does that actually 
make sense to include?
 Should I keep the yielded Section within a Section and if so 
should I document it here? (I included it as sort of an edge case 
but I don't think we want to recommend or encourage that usage.)
 ```

Reply:
Thanks, I'll remove the yielded Section